images/nginx/append/http-log-format.conf

```diff
@@ -0,0 +1,5 @@
+    log_format sdp  '$remote_addr - $remote_user [$time_local] "$request" '
```

Comment:
Creates a new log format called `sdp` based on the `main` template provided by the [Lagoon base image](https://github.com/uselagoon/lagoon-images/blob/main/images/nginx/nginx.conf#L19)